### ISSUE TITLE
Training configuration issues

### ISSUE THREAD

**Issue by niuniu3312** [ljh0v0/D3PM-Pytorch#3]
Firstly, I would like to express my gratitude for providing your code. I would like to inquire about the GPU used during the training process. I used GTX3090 for training, and it took me a lot of time in just one epoch. Additionally, I encountered the following issues during the training process:
Epoch 0:  83%|████████▎ | 196/236 [04:17<00:52,  1.31s/it, loss=0.0211, v_num=5b31]
Validation DataLoader 0:   0%|          | 0/40 [00:00<?, ?it/s]
May I ask how to solve this.
Thanks.
